### PR TITLE
removes ndstore & openconnectome

### PR DESCRIPTION
**Pull request by falkben**
This removes ndstore and openconnectome datasources.  These two services have been offline and will not be reinstated.

### CODE REVIEW

File: src/neuroglancer/datasource/python/backend.ts [hunk 1]
```diff
@@ -21,7 +21,6 @@ import {decodeSkeletonVertexPositionsAndIndices, SkeletonChunk, SkeletonSource}
 import {VertexAttributeInfo} from 'neuroglancer/skeleton/base';
 import {ChunkDecoder} from 'neuroglancer/sliceview/backend_chunk_decoders';
 import {decodeJpegChunk} from 'neuroglancer/sliceview/backend_chunk_decoders/jpeg';
-import {decodeNdstoreNpzChunk} from 'neuroglancer/sliceview/backend_chunk_decoders/ndstoreNpz';
```
Comment (jbms) [inline src/neuroglancer/datasource/python/backend.ts]: This format support shouldn't be removed here.

File: src/neuroglancer/sliceview/backend_chunk_decoders/bossNpz.ts [hunk 2]
```diff
@@ -15,7 +15,7 @@
  */
```
Comment (jbms) [inline src/neuroglancer/sliceview/backend_chunk_decoders/bossNpz.ts]: There isn't really a need to rename this file.

**Comment by jbms** [comment]
Hi,

I apologize for the delay.  Removing these old sources makes sense.  I've just added a few comments regarding the ndstoreNpz.ts file.

File: src/neuroglancer/sliceview/backend_chunk_decoders/bossNpz.ts (hunk 2 above)
Comment (falkben) [reply src/neuroglancer/sliceview/backend_chunk_decoders/bossNpz.ts]: the spec is slightly different between ndstore and boss with the file duplication occurring on the initial BOSS feature add.  sSems easier at this point to just delete the ndstore file?

Comment (jbms) [reply src/neuroglancer/sliceview/backend_chunk_decoders/bossNpz.ts]: Oh, good point.  I didn't realize that.  The Python backend supports multiple channels and therefore needs the ndstore version, though, and they are trivial enough that there may not be a point in trying to combine them.  Perhaps just leave both versions.

**Pull request merged**